docs/tutorials/absorption_in_BLR_with_cubepy.ipynb

```diff
@@ -0,0 +1,175 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import numpy as np\n",
+    "import astropy.units as u\n",
+    "import matplotlib.pyplot as plt\n",
+    "import numpy as np\n",
+    "from agnpy.absorption import Absorption\n",
+    "from agnpy.targets import SphericalShellBLR, lines_dictionary"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "SUM_RATIO_LINES_BLR = 30.652\n",
+    "\n",
+    "def get_absor_blrs(E, L_disk, R_line, r_blob_bh, z):\n",
```

Comment:
this function would be useful to be included in the main code of agnpy
XI_LINE and eps_abs  should be then among the function parameters